libs/sql-schema-describer/src/mysql.rs

```diff
@@ -405,6 +405,8 @@ impl SqlSchemaDescriber {
     fn db_generated(default_string: &str, default_generated: bool) -> DefaultValue {
         if default_generated {
             Self::dbgenerated_expression(default_string)
+        } else if default_string.is_empty() {
+            DefaultValue::db_generated("''")
```

Comment:
I don't know if there is a better way to do this, but here we assume that all empty dbgenerated defaults at this stage in the describer are string defaults. Could there be cases where the column is not text-like and it will be wrong when we use `''` as a default value later?